ci-operator/step-registry/optional-operators/configure-realtime-tuned-profile/optional-operators-configure-realtime-tuned-profile-commands.sh

```diff
@@ -48,8 +48,8 @@ spec:
       energy_perf_bias=performance
       min_perf_pct=100
 
-      [service]
-      service.stalld=start,enable
+      [sysfs]
+      kernel.sched_rt_runtime_us=950000
```

Comment:
is this part of the stalld removal ?

Reply:
Yup, we set it to the default value to be explicit

Reply:
Is this the right place for it ?  should'nt it be under [sysctl] plugin ?
see https://github.com/openshift/release/blob/3843d40e5c4a99a86d30260ed940ce2072db1092/ci-operator/step-registry/optional-operators/configure-realtime-tuned-profile/optional-operators-configure-realtime-tuned-profile-commands.sh#L66-L69

Reply:
Yeah, you're right. Since sysctl is configuring it at runtime, it makes more sense to alter it there. Updated to use `950000` on sysctl